### ISSUE TITLE
Visualization - Add Segmentation - This involves adding graphs and data for various segments in the analysis (e.g. new users, US-only users, etc)

### ISSUE THREAD

**Issue by data-sync-user** [mozilla/experimenter#5559]
**Goal**: Users should be able to view detailed analysis for each segment (e.g. new users, existing users etc.) that was specified in an experiment This is the last scope of work that should be needed before we can turn off partybal.

For each type of data shown on the results page (e.g. summary table, highlights text, weekly tables, etc.), a separate dropdown should be visible to “Show additional segments”. These dropdowns would be hidden by default.

┆Issue is synchronized with this [Jira Epic](https://mozilla-hub.atlassian.net/browse/EXP-840)

**Comment by data-sync-user** [comment]
➤ Kate Hudson commented:

Marina Samuel can you update this epic description with some outcomes/stories? see https://mozilla-hub.atlassian.net/browse/EXP-1610 ( https://mozilla-hub.atlassian.net/browse/EXP-1610|smart-link ) for an example

**Comment by data-sync-user** [comment]
➤ Chelsea Lewis commented:

Moving to Backlog; these are broken down but we are placing on hold until resourcing can be addressed.

**Comment by data-sync-user** [comment]
➤ Jared Lockhart commented:

I’m going to close this and we can refile a new epic/tickets when we have more support from data and can approach segmentation/exposures from a fresh perspective.

**Comment by data-sync-user** [comment]
➤ Shell Escalante commented:

Shell Escalante follow this - as it’s in jetstream and you can see in partyball.  this would be a self service win… but also could be tackled by a looker integration with experiments (as they have detailed segmenting).  May want to rethink the level we address this - build again or is it a good time took at integrating?
